source/api_documentation/signal_api/index.rst

```diff
@@ -346,6 +346,8 @@ If you are unsure about whether you are using this feature, please contact the I
 All Signals and Custom Data within a single request will be grouped together based on the value of the field **partner_unique_id**. There will be a single transaction for each group created.
 These new transactions will have the transaction type *Post Call Event*.
 
+A maximum of 10 transactions can be applied for a single request. However, this limit does not apply to the number of Signals or Custom Data included in a single grouping. See the **Best Practices** section for understanding why this limit is enforced.
```

Comment:
This section is for STR users only. Is there a corresponding note for non-STR users?

Reply:
Yes, for the most part. The Best Practices section discusses the 10 signal limit and assumes that STR is off. So I believe that covers the limit note for non-STR users. 

I assume most of the documentation is written towards non-STR users, and the STR section simply outlines the differences.